tests/core/go_test/fuzz_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build go1.18
+// +build go1.18
+
+package fuzz_test
+
+import "testing"
+
+func Fuzz(f *testing.F) {
+	f.Add("seed")
+	f.Fuzz(func(t *testing.T, s string) {
+		if s != "seed" {
```

Comment:
Is this seed pulled from testdata? Should we add a test for that?

Reply:
That part won't work yet. Seed inputs may be added using `F.Add` or stored in `testdata/fuzz/$targetname`. The testdata path is relative to the test process's current working directory, but that directory might not actually be present. The `data` attribute must cover those files at least, but on Windows, we'll just get a manifest of runfiles, not an actual directory.

One option would be for the generated testmain to wrap each fuzz target with a function that loads inputs from `testdata` with appropriate runfile dereferencing then calls `F.Add`.

Still need to figure where the test can write failing inputs. That's the main thing. Bazel tests generally shouldn't create new files in the source directory.